test/kokoro/grpc_wallet.sh

```diff
@@ -11,7 +11,13 @@ pushd java
 ./gradlew build
 popd
 
-export GOPATH="${HOME}/gopath"
+# Download the latest Go version, extract it under /usr/local (which is the
```

Comment:
Would it make sense to use a directory outside of cloned repo? As I understand this wget is happening in `github/traffic-director-grpc-examples`, see `cd` above.

Reply:
Done. Extracted it into tmpdir.
As a result, i was able to replace the existing multiple build commands with a single one.